packages/MJServer/src/index.ts

```diff
@@ -225,6 +227,63 @@ export const serve = async (resolverPaths: Array<string>, app = createApp(), opt
                                }),
     })
   );
+  
+  // Setup REST API endpoints
+  const authMiddleware = async (req, res, next) => {
+    try {
+      const token = req.headers.authorization?.replace('Bearer ', '');
+      if (!token) {
+        return res.status(401).json({ error: 'Authentication required' });
+      }
```

Comment:
The authorization token doesn't need this preprocessing at all, just send it to `getUserPayload`, as we do elsewhere.